### ISSUE TITLE
inconsistency with R v3.5.1

### ISSUE THREAD

**Issue by fawda123** [ropensci/EML#247]
Hi there, 

I'm attempting to create a simple EML file and am noticing different behavior on different platforms.  My current windows machine runs R3.5.1 and I'm seeing this in the console:

```r
library(EML)
f <- system.file("examples/hf205-methods.md", package = "EML")
set_methods(methods_file = f)
# <methods/>
```

When I run the same on the NCEAS aurora server (R v3.4.4) I get the methods as expected:

```r
f <- system.file("examples/hf205-methods.md", package = "EML")
set_methods(methods_file = f)
# <methods>
#  <methodStep>
#   <description>
#    <section>
#       <title>General Protocols</title>
#      <para>
#    Field methods. All experiments will be carried out in the greenhouse
#    at Harvard Forest. We have developed an instrumentation system that
#   allows us to collect continuous dissolved [O2] measurements:
# etc.
```

As far as I can tell, the only thing that differs between the two systems is the R version and OS.

Mine:
```r
> sessionInfo()
R version 3.5.1 (2018-07-02)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252    LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                           LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] EML_1.0.3

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.18     knitr_1.20       bindr_0.1.1      xml2_1.2.0       magrittr_1.5     tidyselect_0.2.4 uuid_0.1-2      
 [8] R6_2.2.2         rlang_0.2.0      stringr_1.3.1    dplyr_0.7.6      tools_3.5.1      htmltools_0.3.6  yaml_2.1.18     
[15] assertthat_0.2.0 digest_0.6.17    rprojroot_1.3-2  tibble_1.4.2     bindrcpp_0.2.2   purrr_0.2.5      glue_1.3.0      
[22] evaluate_0.10.1  rmarkdown_1.9    stringi_1.1.7    compiler_3.5.1   pillar_1.2.1     backports_1.1.2  pkgconfig_2.0.1
```
NCEAS:
```r
> sessionInfo()
R version 3.4.4 (2018-03-15)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 16.04.5 LTS

Matrix products: default
BLAS: /usr/lib/atlas-base/atlas/libblas.so.3.0
LAPACK: /usr/lib/atlas-base/atlas/liblapack.so.3.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8    LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] EML_1.0.3

loaded via a namespace (and not attached):
[1] compiler_3.4.4 tools_3.4.4    yaml_2.1.18    Rcpp_0.12.15   uuid_0.1-2     xml2_1.2.0    
```

Am I missing something here?

**Comment by cboettig** [comment]
Thanks, can you try the `eml2` branch instead?

**Comment by fawda123** [comment]
Yep, works now.  Sorry for the noise!

**Comment by cboettig** [comment]
Thanks!  Yeah, a few more things to check off and hope to have `eml2` merged into master and and a new `EML` release to go with the soon-to-be-released EML 2.2.0 schema.  (Some backwards compatibility will break, mostly the ugly notations like `stuff@stuff[[1]]@stuff@stuff[[1]]` which will have something more svelte instead :-)

**Comment by fawda123** [comment]
Good stuff, I'm looking to start using these tools more as part of my standard workflow.  Will keep an eye on progress!
